ui/components/Filters.js

```diff
@@ -792,6 +796,14 @@ function MesheryFilters({
     reader.readAsArrayBuffer(file);
   }
 
+  let colViews = [
```

Comment:
@sudhanshutech  Can you move this into `data-table.js`/`constant.js`, since these cols are not specific to Filters but apply to almost all data-tables (including Patterns/Applications).